aws/cloudformation-templates/base/notebook.yaml

```diff
@@ -118,7 +138,7 @@ Resources:
                 Action: 
                   - ssm:PutParameter
                   - ssm:GetParameter
-                Resource: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/retaildemostore-*'
+                Resource: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/retaildemo*'
```

Comment:
We now have two dataset groups now, one for products and one for offers. These differ in their prefix. Rather than increasing the length of the base identifiers which can have knock-on effects, we created two different base identifiers ('_retaildemoproducts-_' and '_retaildemooffers-_' in addition to common resources).

Reply:
Understood on the DSGs. This resource is for SSM parameters which I didn't see the prefix changing elsewhere in the code.

Reply:
Hi again! I double-checked regarding the SSM parameters and you are absolutely right.